core/src/main/scala/kafka/docker/KafkaDockerWrapper.scala

```diff
@@ -52,7 +53,15 @@ object KafkaDockerWrapper extends Logging {
         }
 
         val formatCmd = formatStorageCmd(finalConfigsPath, envVars)
-        StorageTool.main(formatCmd)
+        try {
+          StorageTool.main(formatCmd)
+        } catch {
+          case terseFailure: TerseFailure => if (terseFailure.getMessage.contains(QuorumConfig.QUORUM_VOTERS_CONFIG)) {
```

Comment:
Using string as condition is not stable to me, maybe we should a follow-up to ensure that we catch the "correct" exception.

Reply:
@frankvicky could you please file a minor to address it?

Reply:
sure